### PR TITLE
test createApp process #6

### PR DESCRIPTION
**Pull request by mattvb91**

### CODE REVIEW

File: packages/cli/src/templates/plugin/stencil.config.ts [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+import { Config } from "@stencil/core";
+import { reactOutputTarget } from "@stencil/react-output-target";
+import corejam from "@corejam/dev/dist/rollup";
+import fs from "fs";
+import replace from "@rollup/plugin-replace";
+
+const targets = process.env.targets?.split(",") || [];
+
+const config: Config = {
+  namespace: process.env.NODE_ENV === "production" ? "corejam-plugin-dershop" : "corejam-dev",
```
Comment (patrickhaug) [inline packages/cli/src/templates/plugin/stencil.config.ts:12]: here needs to be a plceholder for namespace, which cli is renaming based on name

**Review by patrickhaug — CHANGES_REQUESTED** [review]
ready to go after updating stencil.config template

File: packages/cli/src/templates/plugin/stencil.config.ts (hunk 1 above)
Comment (mattvb91) [reply packages/cli/src/templates/plugin/stencil.config.ts:12]: Whoops i actually had that in there but in a temp file. Shouldnt a require("package.json").name be ok too?

Comment (patrickhaug) [reply packages/cli/src/templates/plugin/stencil.config.ts:12]: jup sounds good

Comment (mattvb91) [inline packages/cli/src/templates/plugin/stencil.config.ts:12]:
```suggestion
  namespace: process.env.NODE_ENV === "production" ? require("package.json").name : "corejam-dev",
```

**Review by patrickhaug — APPROVED** [review]

**Pull request merged**